rds/variables.tf

```diff
@@ -112,3 +112,15 @@ variable "identifier" {
 locals {
   name = var.name != null ? var.name : "${var.project}-${var.environment}${var.regional ? "-${var.region}" : ""}"
 }
+
+variable "log_min_duration_statement" {
+  type        = number
+  default     = 3000
```

Comment:
default to "-1" https://aws.amazon.com/blogs/database/working-with-rds-and-aurora-postgresql-logs-part-1/